students/seunghun/westagram/urls.py

```diff
@@ -13,7 +13,8 @@
     1. Import the include() function: from django.urls import include, path
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
-from django.urls import path
+from django.urls import path, include
 
 urlpatterns = [
+    path('users/', include('users.urls')),
```

Comment:
```suggestion
    path('users', include('users.urls')),
```
위코드 컨벤션에서는 제일 뒤의 /는 작성하지 않습니다!

Reply:
넵